### PR TITLE
Feature/pelagos 4767 replace router modifier to twig filter

### PR DESCRIPTION
**Pull request by mickel1138**
What still needs to be done is modify all twig files that use URL and PATH, and to determine which ones should be modified. Or we can just do all of them. Suggestions welcome.

### CODE REVIEW

File: src/Util/RouterModifier.php [hunk 1]
```diff
@@ -1,137 +0,0 @@
...
-    }
-
-    /**
-     * Url Matcher.
-     *
-     * @param mixed $pathinfo The path.
-     *
-     * @return array
-     */
-    public function match($pathinfo) : array
```
Comment (praneethpr) [inline src/Util/RouterModifier.php:109]: This exists from UrlMatcherInterface, are we not using this?

[the remark this answers is not in the document]
Comment (mickel1138) [reply src/Util/RouterModifier.php:109]: This is part of the Symfony router, it is not being used in Twig templates by Pelagos.

**Review by praneethpr — APPROVED** [review]

**Review by fightingtexasaggie — COMMENTED** [review]
I noticed we directly have this route in a controller, and /tombstone is a special-case drupal bypass. (like /search, /dif, /data).

src/Controller/UI/TombstoneController.php:     * @Route("/tombstone/{udi}", name="pelagos_app_ui_tombstone_default")

Also, what about all the api routes? They are set up as direct site/api although I suppose we can use pelagos-symfony/api instead. I think we leave in the site/api route just to support external API users, so maybe not an issue.

**Review by fightingtexasaggie — APPROVED** [review]
reviewed

**Pull request merged**